app/typer/miljø.ts

```diff
@@ -0,0 +1,4 @@
+export enum EMiljø {
+  PRODUKSJON = 'PRODUCTION',
+  LOKAL = 'LOCAL',
```

Comment:
```suggestion
  PRODUCTION = 'PRODUCTION',
  LOCAL = 'LOCAL',
```
Det at disse enumsene foreløpig har forskjellige navn på nøkkel og verdi kan være opphav til bugs.

Reply:
Eventuelt kan vi gå for `PRODUKSJON` og `LOKAL`

Reply:
I commit 6a02da5 gjekk eg for Norsk. Noen tanker der? @halvorbmundal / @daphneleebeek 
Tenker litt i forhold til Nav sin kode, kontra den generelle standarden i `.env` 
( Om eg har forstått det riktig er det engelsk med lov til å ha forkorta navn på variabler )

Reply:
Begge deler går bra for min del 👍